src/appleseed/renderer/modeling/scene/objectinstance.h

```diff
@@ -170,6 +176,26 @@ class APPLESEED_DLLSYMBOL ObjectInstance
     // Return the object bound to this instance.
     Object& get_object() const;
 
+    // SSS set
+    // By default, each object instance belongs to its own SSS set
+    struct SubsurfaceScatteringSet
```

Comment:
I suggest moving this entire class to the .cpp file. It doesn't need to be exposed.

Even better, I think you can completely get rid of this class and just use bare `OIIO::ustring` objects stored in object instances, with the convention that an empty ustring means that the object instance is in its own SSS set.